src/txmempool.cpp

```diff
@@ -395,7 +395,7 @@ bool CTxMemPool::addUnchecked(const uint256& hash, const CTxMemPoolEntry &entry,
 {
     NotifyEntryAdded(entry.GetSharedTx());
     // Add to memory pool without checking anything.
-    // Used by main.cpp AcceptToMemoryPool(), which DOES do
+    // Used by validation.cpp AcceptToMemoryPool(), which DOES do
```

Comment:
It's useful to know where the transaction is being checked before it is added using this function. Otherwise I guess you could use grep to figure out where AcceptToMemoryPool() went but why not just say where it is?